packages/yoga/src/Avatar/native/Avatar.jsx

```diff
@@ -26,29 +39,51 @@ const Content = ({ icon, src, fill, content, stroke }) => {
  * It has two shapes: default and circle
  */
 const Avatar = forwardRef(
-  ({ icon = BuildingFilled, src, children, fill, stroke, ...props }, ref) => {
+  (
+    {
+      icon = BuildingFilled,
+      src,
+      children,
+      fill,
+      stroke,
+      borderRadius,
+      width,
+      height,
```

Comment:
Maybe we can move the default value to the prop, just like we did to the icon.

```suggestion
      borderRadius = 'small',
      width = 48,
      height = 48,
```